apps/common/src/main/resources/db/migration/canton-network/postgres/stable/V043__scan_verdict_store.sql

```diff
@@ -0,0 +1,55 @@
+-- Table 1: scan_verdict_store
```

Comment:
nitpick: I would remove the `table 1:` and `table 2:` from these comments

Reply:
Fixed